src/main/java/net/sf/jabref/JabRef.java

```diff
@@ -575,7 +196,7 @@ private void setLookAndFeel() {
         }
     }
 
-    private void openWindow(List<ParserResult> loaded) {
+    private void openWindow(List<ParserResult> loaded, boolean isBlank) {
```

Comment:
We could move this method and other code to a JabRefGUI class which then loads the gui, sets the look and feel, etc.